bundles/org.openhab.binding.ecovacs/src/main/resources/OH-INF/i18n/ecovacs.properties

```diff
@@ -0,0 +1,173 @@
+# binding
+binding.ecovacs.name = Ecovacs Binding
+binding.ecovacs.description = This is the binding for Ecovacs Deebot vacuum cleaners.
```

Comment:
This needs to be regenerated after migrating to addon.xml.

Reply:
Done ... strangely enough, `mvn i18n:generate-default-translations` still adds the 'binding...' texts, though.

Reply:
Do you still have `binding.xml` locally?

Reply:
Do you mean it adds them or that it doesn't remove them?

Reply:
> Do you still have binding.xml locally?

Nope:

```
find src/main/resources/OH-INF
src/main/resources/OH-INF
src/main/resources/OH-INF/i18n
src/main/resources/OH-INF/i18n/ecovacs.properties
src/main/resources/OH-INF/thing
src/main/resources/OH-INF/thing/bridge.xml
src/main/resources/OH-INF/thing/thing-types.xml
src/main/resources/OH-INF/addon
src/main/resources/OH-INF/addon/addon.xml
```

> Do you mean it adds them or that it doesn't remove them?

The former: I first manually converted lines 2 and 3 to `addon.ecovacs...`. After running `mvn i18n:generate-default-translations`, I suddenly had both.

Reply:
I now manually deleted them, and the tool keeps them deleted. Somewhat strange, but whatever...